services/src/test/scala/cromwell/pipeline/service/GitLabProjectVersioningTest.scala

```diff
@@ -38,19 +39,50 @@ class GitLabProjectVersioningTest extends AsyncWordSpec with ScalaFutures with M
           _ shouldBe Right(project)
         }
       }
-      "throw new VersioningException with not 400 response" taggedAs Service in {
+      "throw new VersioningException with 400 response" taggedAs Service in {
         when(request(activeProject)).thenReturn(Future.successful(Response(StatusCodes.BadRequest.intValue, "", Map())))
         gitLabProjectVersioning.createRepository(activeProject).map {
           _ shouldBe Left(VersioningException("The repository was not created. Response status: 400"))
         }
       }
     }
+
+    "getProjectVersions" should {
+      def dummyVersionsJson: String = s"[${Json.stringify(Json.toJson(dummyVersion))}]"
```

Comment:
Why def?

Reply:
> Why def?

why don't you like defs ?)

Reply:
Cause we can crush on the first test, and we will not reach this value, so I think it's better to call a method and do not have a val in memory.

Reply:
Than lazy val is more fitting for this

Reply:
Deal! Fixed.